bin/portal-bridge/src/bridge/e2hs.rs

```diff
@@ -111,7 +111,18 @@ impl E2HSBridge {
     pub async fn launch(&self) {
         info!("Launching E2HS bridge");
 
-        let epochs = block_range_to_epochs(self.block_range.start, self.block_range.end);
+        let block_range_end = match self.block_range.end {
+            BlockRangeEnd::Latest => {
+                let max_epoch = self
+                    .e2hs_files
+                    .keys()
+                    .max()
+                    .expect("to be able to get max epoch");
+                max_epoch * BLOCKS_PER_E2HS as u64 + BLOCKS_PER_E2HS as u64
```

Comment:
```suggestion
                max_epoch * BLOCKS_PER_E2HS as u64 + BLOCKS_PER_E2HS as u64 - 1
```

My understanding is that end of the range is inclusive, so it should be like I suggested, right?

You can also do it this way:
```suggestion
                (max_epoch + 1) * BLOCKS_PER_E2HS as u64 - 1
```